README.md

```diff
@@ -269,6 +269,21 @@ export default class App {
 }
 ```
 
+#### Decorators
+
+The `@provide` decorator lets you inject a Redux instance more succinctly:
```

Comment:
uses `more succinctly` here and `less verbosely` above. Probably best to keep consistent language.  I like more `more succinctly`, personally, but I think the connect decorator language should be changed to match.